code/modules/mob/living/silicon/robot/life.dm

```diff
@@ -298,20 +285,43 @@
 		if(client && !client.adminobs)
 			reset_view(null)
 
+	if(emagged)
+		throw_alert("hacked", /obj/screen/alert/hacked)

```

Comment:
The intent of this alert is to constantly display during the duration of the robot being hacked. Moving it anywhere else would cause the alert not to show with things like "new client gets this robot body" or "client logs out and logs back in."

Reply:
As answered by Shadow, this is intended to stay as it is.